app/models/search_builder.rb

```diff
@@ -125,7 +125,7 @@ def modify_combo_param_with_absent_q(solr_parameters)
     if !blacklight_params[:q].present?
       if blacklight_params[:f].present?
         # we have user filters, so avoid NPE by ignoring q in combo domain
-        solr_parameters['combo'] = '{!filters param=$fq excludeTags=cluster}'
+        solr_parameters['combo'] = '{!filters param=$fq excludeTags=cluster,no_correlation}' # NOTE: $correlation_domain is applied within facets
```

Comment:
so $correlation_domain is applied within facets, and by excluding this tag here we are bypassing that?

Reply:
Good catch; see: https://github.com/upenn-libraries/discovery-app/blob/c3882d5be20e6558c245f00d2268a17bcd2466a7/app/controllers/catalog_controller.rb#L171-L173

Reply:
Not bypassing anything at the moment, but we can safely (and in fact _want_ to) exclude certain filters from calculation; not doing it yet, but we can/should/will ...